<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBoxLineRotation</class>
 <widget class="QDialog" name="DialogBoxLineRotation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Line Rotation</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRotate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDX">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>dX</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDZ">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDY">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxBX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>69</width>
     <height>26</height>
    </size>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>dZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bY</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxRotationAngle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>dY</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxBY">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotation Angle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxBZ">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
